html-css/override-extend-variables/step1-instructions.md

```diff
@@ -1,11 +1,11 @@
-## Add code to the workspace
-
-Before we get started, let's add some code to the workspace that we can modify.
-
+## Add code to the workspace
+ 
+Before you get started, let's add some code to the workspace that you can modify.
+ 
 1) <strong>Copy code into the index.html file.</strong>
-
-Click the <strong>Copy to Editor</strong> button below to add html for a card to the `index.html` file.
-
+ 
+2) <strong> Click the Copy to Editor</strong> button below to add html for a card to the `index.html` file.
```

Comment:
The entire sentence should be in bold

Reply:
<strong> Click the Copy to Editor</strong> button below to add html for a card to the `index.html` file.
sb
<strong> Click the Copy to Editor button below to add html for a card to the `index.html` file.</strong>